src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
+$color: #eee;
+$cell-size: 100px;
+$gap: 1px;
+$calendarWidth: 726px;
```

Comment:
The variable name '$calendarWidth' uses camelCase, which is not consistent with the other variable names that use kebab-case (e.g., '$cell-size'). For consistency, consider renaming it to '$calendar-width'.